screens/LoginScreen.js

```diff
@@ -14,34 +14,97 @@ import {
     View
 } from 'native-base';
 
+import { ErrorMessage } from '../components/ErrorMessage';
+import { baseUrl } from '../constants/api';
+
 export default class LoginScreen extends React.Component {
     static navigationOptions = {
         headerTitle: 'Emote',
         headerBackTitle: 'Back'
-    }
+    };
 
-    submitForm = () => {
-        // Validate form has proper input
+    state = {
+        email: '',
+        password: '',
+        errorMessage: ''
+    };
 
-        // Submit credentials
+    submitForm = async () => {
+        const { email, password } = this.state;
+
+        // Validate form has proper fields
+        if (!email || !password) {
+            this.setState({
+                errorMessage:
+                    'You must enter your Email and Password.'
+            });
+            return;
+        }
+        const data = {
+            email,
+            password
+        };
 
-        // In promise response, if validated, direct to main page
-        this.props.navigation.navigate('Main');
+        // Submit credentials
+        fetch(`${baseUrl}login`, {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(data)
+        })
+            .then((response) => {
+                if (response.ok) {
+                    return response.json();
+                } else {
+                    throw Error(
+                        'There was an error, please try again.'
```

Comment:
had trouble getting at the error message from the API, so just set the message here for now. fetch can't read the response unless you call response.json() on it.